### PR TITLE
Advisor contract (Cameligo)

### PR DESCRIPTION
**Pull request by frankhillard**

### CODE REVIEW

**Review by oteku — APPROVED** [review]
LGTM

File: test/ligo/test.mligo [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    let (address_contract, code_contract, _) = Test.originate_from_file file_path mainName views storage 0tez in
+    let taddress_contract = (Test.cast_address address_contract : (p, s) typed_address) in
+    address_contract, taddress_contract, Test.to_contract taddress_contract
+
+let _test =
+  // deploy INDICE smart contract 
+  let indice_initial_storage = 4 in
+  let () = Test.log("deploy INDICE smart contract") in
+  let iis = Test.run (fun (x:INDICE.storage) -> x) indice_initial_storage in
+  // path relative to where the command run test in runned
```
Comment (d-plaindoux) [inline test/ligo/test.mligo]: This code (if not used) should be removed

File: test/ligo/test.mligo [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  // deploy ADVISOR contract 
+  let () = Test.log("deploy ADVISOR smart contract") in
+  let advisor_initial_storage : ADVISOR.storage = {
+    indiceAddress=address_indice; 
+    algorithm=(fun(i : int) -> if i < 10 then True else False); 
+    result=False;
+    metadata=(Big_map.empty: (string, bytes) big_map);
+  } in
+  let ais = Test.run (fun (x:ADVISOR.storage) -> x) advisor_initial_storage in
+  //let advisor_contract_path = "advisor.mligo" in //"views_hangzhou/cameligo/advisor.mligo" in
```
Comment (d-plaindoux) [inline test/ligo/test.mligo]: This code (if not used) should be removed

File: test/ligo/test2.mligo [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    address_contract, taddress_contract, Test.to_contract taddress_contract
+
+let test =
+  
+  // deploy DUMMY smart contract 
+  let indice_initial_storage : DUMMY.indiceStorage = 4 in
+  let () = Test.log("deploy DUMMY smart contract") in
+  // transpile storage in michelson code
+  let iis = Test.run (fun (x:DUMMY.indiceStorage) -> x) indice_initial_storage in
+  //let indice_contract_path = "test/ligo/indice_no_view.mligo" in
```
Comment (d-plaindoux) [inline test/ligo/test2.mligo]: This code (if not used) should be removed

File: test/ligo/test2.mligo [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  let () = Test.log("deploy ADVISOR smart contract") in
+  let advisor_initial_storage : ADVISOR.storage = {
+    indiceAddress=address_indice;
+    algorithm=(fun(i : int) -> if i < 10 then True else False);
+    result=False;
+    metadata=(Big_map.empty: (string, bytes) big_map);
+  } in
+  // transpile storage in michelson code
+  let ais = Test.run (fun (x:ADVISOR.storage) -> x) advisor_initial_storage in
+  //let advisor_contract_path = "advisor.mligo" in //"views_hangzhou/cameligo/advisor.mligo" in
```
Comment (d-plaindoux) [inline test/ligo/test2.mligo]: This code (if not used) should be removed

**Review by d-plaindoux — APPROVED** [review]

**Pull request merged**